### PR TITLE
Updated events

### PR DESCRIPTION
**Pull request by lboyette-okta**
I've created a parser for our message properties file to update our events on developer.okta.com.

A few notes: the tables generated are fixed width. Some of the event names are very long, so I encoded the periods with zero-width spaces to allow any browser to wrap the names on periods.

@karlmcguinness-okta @stsai-okta @eklein-okta @croche-okta @rwang-okta @tthompson-okta @larsjohansen-okta

### CODE REVIEW

**Comment by croche-okta** [comment]
Description for ``app.​ad.​group.​provision.​updateMembershipError`` has special characters in it.

**Comment by lboyette-okta** [comment]
screen shot 2015-02-11 at 4 49 55 pm

File: _tools/event_generator.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+current_key = ""
+current_value = ""
+events_started = false
+first_table = true
+File.foreach(source, :encoding => 'iso-8859-1:utf-8') do |line|
+	# Check for headings
+	if /#\s*[A-z]+[-\s\.A-z0-9]*$/ =~ line
+    events_started = true
+
+		heading = line[/[a-zA-Z]+[-\s\.a-zA-Z=0-9]*$/]
```
Comment (stsai-okta) [inline _tools/event_generator.rb]: your indentation is screwed up

**Comment by mystiberry-okta** [comment]
@lboyette-okta What's the status on the conflicts here? I can't merge until the conflicts are resolved. We have a bit of a backlog in merge requests, so I'm following up on all of them. Thanks!

**Comment by mystiberry-okta** [comment]
oops, didn't notice this was Feb of *last year*. Is this still valid?

**Comment by karlmcguinness-okta** [comment]
We need a solution for this still, it was never productized.  I will close this PR and start a thread

**Pull request closed without merging**